src/main/java/Reporter.java

```diff
@@ -0,0 +1,48 @@
+import java.util.stream.Collectors;
+
+class Reporter {
+
+    private static final String DELIMITER = "-";
+
+    private static void printDelimiter() {
+        System.out.println(DELIMITER.repeat(30));
+    }
+
+    public static void getFullReport(Calculator calculator) {
+        System.out.println("[[[\t\tОтчет\t\t]]]");
+        printDelimiter();
+
+        System.out.println("Добавленные товары:");
+        System.out.println(
+                calculator.getItems().stream().map(Item::toString)
+                        .collect(Collectors.joining("\n"))
+        );
+        printDelimiter();
+
+        double amountPerPerson = calculator.getPerPersonAmount();
+
+        System.out.printf(
+                "\nКаждый из вас должен заплатить по %.2f %s\n",
```

Comment:
🍏 
```suggestion
                "\nКаждый из вас должен заплатить %.2f %s\n",
```
Иначе звучит странно: "Каждый из вас должен заплатить по 1 рубль"